Formula/fff.rb

```diff
@@ -0,0 +1,18 @@
+class Fff < Formula
+  desc ":rocket: fucking fast file-manager"
+  homepage "https://github.com/dylanaraps/fff"
+  url "https://github.com/dylanaraps/fff/archive/1.0.tar.gz"
+  sha256 "6e5eed18b50b3e1a2497092e618665ff00b71b27b4f7653f9693ee40517d1a8a"
+
+  depends_on "bash"
+  depends_on "coreutils"
```

Comment:
Does it not work with macOS system tools?

Reply:
It should. All it needs is `bash 3.2+` and POSIX compliant `coreutils`.

Reply:
Does depends_on not check for the required formula before attempting to install? 

If not, I can remove lines 7 and 8

Reply:
@lougehrig10 You’re correct in that `depends_on` does ensure the required formula is installed.

The point being made by @fxcoudert here is that Homebrew’s `bash` formula provides Bash 4. That would be overly strict here because `/bin/bash`, which ships with macOS, is already guaranteed to be Bash 3.2.x.

Similarly, the `coreutils` formula provides GNU coreutils though that’s probably unnecessary here, too. The use case for `coreutils` (the Homebrew formula) is when your program explicitly needs GNU coreutils, i. e. some GNU-specific part of that.

We should lose both line 7 and 8 if you ask me.